examples/simple/main.tf

```diff
@@ -14,22 +14,6 @@
  * limitations under the License.
  */
 
-provider "google" {
```

Comment:
Same as other comment, we should specify these to ensure tests are reliable. If they need to updated, lets do that.